PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/Endpoint/configuration_classes/pcie_capability.cpp

```diff
@@ -0,0 +1,27 @@
+#include "pcie_capability.h"
+
+class ConfigurationVisitor;
+
+PCIECapability * PCIECapability::pcieCapability = nullptr;
+
+PCIECapability::PCIECapability()
+{
+}
+
+PCIECapability::~PCIECapability()
+{
+}
+
+PCIECapability* PCIECapability::constructPCIECapability()
+{
+    if(pcieCapability == nullptr)
+        pcieCapability = new PCIECapability();
+
+    return pcieCapability;
+}
+
+unsigned int PCIECapability::accept(shared_ptr<ConfigurationVisitor> visitor)
+{
+    //return visitor->visitPcieCapabilityStructure(this);
```

Comment:
I see here it misses the implementation

Reply:
Same the above comment regarding the same situation.